src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceFileLogger.cs

```diff
@@ -87,7 +89,11 @@ public virtual async Task ProcessLogQueue(object state)
             while (!_cancellationTokenSource.IsCancellationRequested)
             {
                 await InternalProcessLogQueue();
-                await Task.Delay(TimeSpan.FromSeconds(FlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                await Task.Delay(TimeSpan.FromSeconds(StartingFlushFrequencySeconds), _cancellationTokenSource.Token).ContinueWith(task => { });
+                if ( StartingFlushFrequencySeconds < MaxFlushFrequencySeconds)
```

Comment:
@divyagandhisethi - please review -  also do share how is this handled in Linux Consumption as it is file based logging as well.

Reply:
Discussed this with Divya, Linux Consumption does not use File Logging